sfreeley/front-end-capstone
# Sf put post

# Description
Fixes 4, 5, 7, 8, 9

1. Full functional CRUD for medication list and medication history list
2. User can post from home page and medication list 
3. User can edit from medication, history list, and details list
4. User can delete from medication, history, and details list
5. User can check box on medication card and will edit "taking" property to false and send card to medication history
6.User can do the opposite of above in medication history
7. User can also send medication card to medication history from details view
7.Drug resources page available
8. Search functionality is able to filter properly and display correct search with onChange instead of onClick, BUT when user backspaces or clears the input search field it will display all the medication cards

## Type of change

# Testing Instructions
1. npm start in terminal
2. json-server -p 5002 -w database.json
3. click all details, edit, and delete buttons and save to history/medication list checkbox buttons on medication card to see full CRUD functionality (edit drug will be in a modal for medication list, but will be a new view if edited from details or medication history

# Checklist:
